app/controllers/hosts_controller.rb

```diff
@@ -869,4 +910,16 @@ def validate_multiple_puppet_proxy
       redirect_to(select_multiple_puppet_proxy_hosts_path) and return false
     end
   end
+
+  def validate_multiple_puppet_ca_proxy
```

Comment:
Ditto, identical logic to that of `validate_multiple_puppet_proxy`, please refactor